neural_compressor/torch/utils/utility.py

```diff
@@ -630,3 +631,66 @@ def run_fn_for_vlm_autoround(model, dataloader, seqlen=512, nsamples=512):  # pr
         total_cnt += input_ids.shape[0] if len(input_ids.shape) > 1 else 1
         if total_cnt >= nsamples:
             break
+
+
+def can_pack_with_numba():  # pragma: no cover
```

Comment:
How about moving these to `environ.py`

Reply:
Done in [a292660](https://github.com/intel/neural-compressor/pull/2068/commits/a292660f4b5e546c9b60af61ef432f6beefb638c)